web/src/beta/lib/core/StoryPanel/Page/index.tsx

```diff
@@ -47,82 +71,137 @@ const StoryPage: React.FC<Props> = ({
   onBlockDelete,
   onBlockSelect,
   onPropertyUpdate,
+  handleMoveBlock,
```

Comment:
onBlockMove

Reply:
@KaWaite  handleMoveBlock function is used to update MoveStoryBlock mutation but I think this one (onBlockMove) has other functionalities.

Reply:
Sorry don't quite understand what you mean.
For me, I was just pointing out the naming convention is wrong. For component props, functions should be named `onNounVerb`, whereas locally defined functions should be `handleNounVerb`.
This helps with readability and is a React standard 🙏🏻

Reply:
@KaWaite there is already onBlockMove function so it will be a duplicate may be I can make it onBockDnd or onBlockMoved